core/src/main/scala/org/apache/spark/eventhubs/package.scala

```diff
@@ -39,6 +39,7 @@ package object eventhubs {
   val DefaultFailOnDataLoss = "true"
   val DefaultUseSimulatedClient = "false"
   val StartingSequenceNumber = 0L
+  val DefaultEpoch = 0L
```

Comment:
tip: use a small & unique number like 1761 
 - from the error messages you can tell - another spark job trumped the receiver